kimchi/src/circuits/berkeley_columns.rs

```diff
@@ -67,6 +67,7 @@ impl<'a> super::expr::AlphaChallengeTerm<'a> for BerkeleyChallengeTerm {
     const ALPHA: Self = Self::Alpha;
 }
 
+#[derive(Clone)]
```

Comment:
Remove.  Was originally to work around that I didn't want to borrow the `Environment` in an initial implementation.  But that has been fixed, so this no longer serves any meaningful purpose.